db/test/consul_test.go

```diff
@@ -0,0 +1,62 @@
+package test
+
+import (
+	"os/exec"
+	"time"
+
+	. "gopkg.in/check.v1"
+
+	"github.com/contiv/volplugin/db"
+	"github.com/contiv/volplugin/db/impl/consul"
+	"github.com/hashicorp/consul/api"
+)
+
+func (s *testSuite) TestNewConsulClient(c *C) {
+	if !IsConsul() {
+		c.Skip("not consul")
+	}
+
+	client, err := getConsulClient()
+	c.Assert(err, IsNil)
+
+	volClient, err := consul.NewClient("volplugin", &api.Config{})
```

Comment:
Why are these three lines duplicated two other times in this test? (one with some better error formatting but identical otherwise)